tests/ui/traits/next-solver/global-cache-and-parallel-frontend.rs

```diff
@@ -0,0 +1,15 @@
+//@ compile-flags: -Zthreads=16
+
```

Comment:
please add a comment explaining what this tests (and link to the original issue)